android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1613,6 +1620,22 @@ public int getKeyboardWidth() {
       return kbWidth;
     }
 
+    // This annotation is required in Jelly Bean and later:
+    @JavascriptInterface
+    public void beepKeyboard() {
+      int duration = 500; // milliseconds
```

Comment:
This is definitely too long. When I tested it was quite dramatic :) I tried with 100ms and that felt more appropriate. It needs to be a nudge, not a big serious flag. Can you experiment with durations <100ms to see how they feel as well? What do you think?

Reply:
I tried 50ms and 100ms and can't really tell the difference. Will stick with 100ms